README.md

```diff
@@ -120,6 +126,9 @@ git submodule update --init --recursive
 set PATH=<protobuf_install_dir>\bin;%PATH%
```

Comment:
I think \include and \libs also need to be added to the path

Reply:
Only including the \bin path which has `protoc.exe` works in my local Windows... Are \include and \libs needed for building ONNX with static protobuf in GitHub Action CI?

Reply:
yes they are needed. Please add them

Reply:
Added. Thanks